page-factory-core/src/main/java/ru/sbtqa/tag/stepdefs/en/CoreSteps.java

```diff
@@ -9,7 +9,7 @@
 import ru.sbtqa.tag.pagefactory.exceptions.PageInitializationException;
 import ru.sbtqa.tag.stepdefs.CoreGenericSteps;
 
-public class CoreStepDefs extends CoreGenericSteps {
+public class CoreSteps extends CoreGenericSteps {
 
     @Before(order = 0)
     public void preSetUp(Scenario scenario) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add the "@Override" annotation above this method signature ([squid:S1161](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1161))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=70&issues=AWSwZVrXIUupByjQG-e8&open=AWSwZVrXIUupByjQG-e8)

[ISSUE_KEY]: AWSwZVrXIUupByjQG-e8